aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/Orchestrator.java

```diff
@@ -97,6 +101,9 @@ public Orchestrator(
         Merger merger = new Merger(mutationOutbox, versionRepository, localStorageAdapter);
         SyncTimeRegistry syncTimeRegistry = new SyncTimeRegistry(localStorageAdapter);
 
+        this.localStorageAdapter = localStorageAdapter;
```

Comment:
Is this dead store, too?

Reply:
Yup...can't remember why, but I think I was using it somewhere other than the constructor in one of my previous interations.